tests/page-objects/standard/forms/delivery.wdio.page.js

```diff
@@ -0,0 +1,74 @@
+const OUTCOME = {liveBirth: 'healthy', stillBirth: 'still_birth', miscarriage: 'miscarriage'};
+const LOCATION = {facility: 'f', homeAttendant: 's', homeNoAttendant: 'ns'};
+
+const FORM = 'form[data-form-id="delivery"]';
+const pregnancyOutcome = (value) => $(`${FORM} ` +
+  `input[name="/delivery/group_delivery_summary/g_pregnancy_outcome"][value="${value}"`);
+const pregnancyOutcomeLabel = (value) => $(`${FORM} ` +
+  `span[data-itext-id="/delivery/group_delivery_summary/g_pregnancy_outcome/${value}:label"]`);
+const deliveryLocation = (value) => $(`${FORM} ` +
+  `input[name="/delivery/group_delivery_summary/g_delivery_code"][value="${value}"`);
+const deliveryLocationLabel = (value) => $(`${FORM} ` +
+  `span[data-itext-id="/delivery/group_delivery_summary/g_delivery_code/${value}:label"]`);
+const deliveryDate = () => $(`${FORM} div.widget.date input`);
+const smsNote = () => $(`${FORM} textarea[name="/delivery/group_note/g_chw_sms"]`);
+const outcomeSummary = () => $(`${FORM} ` + 
+  `span[data-value=" /delivery/group_delivery_summary/display_delivery_outcome "]`);
+const locationSummary = () => $(`${FORM} span[data-value=" /delivery/group_summary/r_delivery_location "]`);
+const followUpSMS = () => $(`${FORM} span[data-value=" /delivery/chw_sms "]`);
+
+const selectPregnancyOutcome = async (value = OUTCOME.liveBirth) => {
+  const outcome = await pregnancyOutcome(value);
+  await outcome.waitForDisplayed();
+  await outcome.click();
+  return await (await pregnancyOutcomeLabel(value)).getText();
```

Comment:
```suggestion
return (await pregnancyOutcomeLabel(value)).getText();
```

Minor, but I don't think you actually need this outer `await`.  Since you are returning a value from an `async` method, I think it is okay to just return a promise (without awaiting it).  

Might be worth trying it.  If it does not work, just ignore this comment. If it does work, you could simplify this code (and the other similar functions below).

Reply:
You are completely right, it worked!
Thanks